infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/collections/impl/SszMutablePrimitiveListImpl.java

```diff
@@ -76,6 +77,19 @@ public void set(int index, SszElementT value) {
     }
   }
 
+  @Override
+  @SuppressWarnings("unchecked")
+  public void setAll(final SszElementT value, final int startIndex, final int endIndex) {
+    if (startIndex == 0 && endIndex >= size()) {
+      // We're replacing everything so just build a new backing tree rather that iterating
+      setImmutableBackingData(
+          (AbstractSszComposite<SszElementT>) getSchema().nCopies(value, endIndex));
+      cachedSize = endIndex;
+      return;
+    }
+    SszMutablePrimitiveList.super.setAll(value, startIndex, endIndex);
```

Comment:
```suggestion
    } esle {
        SszMutablePrimitiveList.super.setAll(value, startIndex, endIndex);
    }
```
nit: just a minor style suggestion